### PR TITLE
Pipepline example

### PR DESCRIPTION
**Pull request by evgeny-chufarov**
PR adds the pipeline example.

### CODE REVIEW

File: roles/pipeline/pipeline_example/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (alexander-demicev) [inline roles/pipeline/pipeline_example/defaults/main.yml]: Can you remove the empty line at beginning?

**Review by alexander-demicev — CHANGES_REQUESTED** [review]
You should also add this role to all.yml, otherwise LGTM

File: roles/pipeline/pipeline_example/defaults/main.yml (hunk 1 above)
Comment (evgeny-chufarov) [reply roles/pipeline/pipeline_example/defaults/main.yml]: done

File: roles/pipeline/pipeline_example/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      - app = pipeline
+  register: buildconfig
+  until: buildconfig
+         and (buildconfig.resources | length > 0)
+  retries: 10
+  delay: 5
+
+- name: Create a backup
+  block:
+    - name: Create backup
```
Comment (alexander-demicev) [inline roles/pipeline/pipeline_example/tasks/main.yml:25]: You would need when: .. here

File: roles/pipeline/pipeline_example/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      include_role:
+        name: backup
+
+- name: Restore pipeline
+  when: not travisCI
```
Comment (alexander-demicev) [inline roles/pipeline/pipeline_example/tasks/main.yml]: There is no travisCI var now, check similar playbooks

**Review by alexander-demicev — CHANGES_REQUESTED** [review]

**Review by alexander-demicev — APPROVED** [review]

**Pull request merged**